configure.ac

```diff
@@ -53,6 +53,16 @@ AC_DEFUN([WITH_SAVE_ENV],
 [RESTORE_ENV]
 )dnl
 
+dnl AC_DEFINE*() wrappers.
+AC_DEFUN([PY_DEFINE], [dnl
+  AC_DEFINE([[Py_]$1], [$2], [$3])dnl
+  AC_DEFINE([$1], [$2], [$3])dnl
```

Comment:
If you change the `Py` macro for whatever reason, the non-prefixed `Py_` macro would not be changed. Instead, it should be aliased to `[Py_]#1` directly.